docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -53,6 +53,35 @@ updates.
 
 ## Enroll instructions
 
+<Details
+scope={["cloud"]}
```

Comment:
Why is this only visible for the `cloud` scope? 

It's also worth noting that most users tend not to use the scope switcher when they navigate between pages, so there's a good chance that users will be viewing the default (`oss`) scope and not see this `Details` box.

Finally, if this is a flow we expect most users to use, I would remove it from the `Details` box and make it part of the main body text.

Reply:
This feature is currently only available on cloud.  This will eventually be able to be moved into the main docs, but we need to do some performance improvements first so that we can enable the feature by default.  Until then, we have to scope this to cloud to avoid confusion.

We also suggest these commands in the alert so that users will be informed of this strategy even if they miss the docs.